evap/staff/tests/test_views.py

```diff
@@ -915,6 +913,10 @@ def setUpTestData(cls):
         # This is necessary so that the call to is_single_result does not fail.
         responsible = mommy.make(UserProfile)
         cls.contribution = mommy.make(Contribution, course=cls.course, contributor=responsible, responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+        cls.url = reverse('staff:course_edit', args=[cls.course.semester.pk, cls.course.pk])
```

Comment:
that's not a bad idea in general, but right now every view testcase has this `url` class variable by convention. i'm not 100% sure that's the best we can do, but would stick to it for consistency until we make a different decision.